0-ad-history-mod-package/simulation/templates/units/han/siege_mangonel_packed.xml

```diff
@@ -5,6 +5,11 @@
     <GenericName>Mangonel</GenericName>
     <SpecificName>Tóushí Chē</SpecificName>
     <Icon>units/han/siege_mangonel_packed.png</Icon>
+    <History>
+        Unlike ancient Western civilizations, the ancient Chinese did not use tension or torsion mechanisms for catapults. Instead, their catapults were man-powered: A long horizontal beam was attached to a hinge that was mounted on a framework. A sling was fastened to the end of the beam's back arm. Ropes hang down from the (shorter) front arm or from a short cross beam mounted there. The team of men operating the catapult stood below the shorter arm. On command, they all simultaneously pulled down the ropes, causing the beam to rotate, the sling to open, and whatever projectile was placed in the sling (usually a stone) to be hurled forward. Such catapults are called mangonels, or traction trebuchets.
+        When mangonels in the early Middle Ages also found their way from China to Europe, they proved to be superior to the traditional catapults there, powered by the tension of a bow or the torsion of sinew bundles. This was for several reasons: Firstly, they were simpler to construct and operate. Secondly, because of their mechanical simplicity they could be reloaded faster and fire at a higher rate. For the same reason, they were also way more reliable than the traditional torsion artillery. For example, in contrast to those, a mangonel's performance was not affected by damp weather conditions. Lastly, it could easily be scaled up or down for different needs by increasing or decreasing the number of men pulling the ropes. All in all, mangonels were more user-friendly, so to speak.
```

Comment:
I simplified this paragraph to make it cleaner, and I think it means the same thing. If I missed something, feel free to change it.

```suggestion
        In the early Middle Ages the mangonels made their way to Europe,  when Improved with the tension of a bow or a torsion of sinew bundles, they proved to be superior to traditional catapults. Because they were simpler, more reliable, and could fire at a faster rate. The mangonels could easily be scaled larger or smaller for different needs.
```